types/jasmine/index.d.ts

```diff
@@ -1164,6 +1165,75 @@ declare namespace jasmine {
     var MAX_PRETTY_PRINT_DEPTH: number;
 
     var version: string;
+
+    interface JasmineOptions {
+        /**
+         * The path to the project's base directory. This can be absolute or relative
+         * to the current working directory. If it isn't specified, the current working
+         * directory will be used.
+         */
+        projectBaseDir?: string;
+    }
+
+    interface JasmineConfig {
```

Comment:
Where is `seed` prop?
https://github.com/jasmine/jasmine/blob/163f93d6ffe905a7d2f907f83cea934706b69e64/src/core/Env.js#L61

Reply:
This interface is for jasmine-npm `loadConfig` method not `env.configure`

[docs](https://jasmine.github.io/api/npm/edge/Configuration.html)

Reply:
Really sry 🤕